dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -590,16 +587,37 @@ public static <T> T getArbitraryURL(String url, GenericType<T> type, ApiClient c
      * @param resourceFileName
      * @return
      */
-    public static File getUniversalResourceFile(String resourceFileName) {
-        File tempResourceFile;
+    public static Optional<File> getUniversalResourceFile(String resourceFileName) {
+        File tempResourceFile = null;
         try (InputStream inputStream = Objects.requireNonNull(CommonTestUtilities.class.getClassLoader().getResourceAsStream(resourceFileName))) {
             tempResourceFile = File.createTempFile(resourceFileName, null);
             FileUtils.copyInputStreamToFile(inputStream, tempResourceFile);
-        } catch (IOException e) {
-            LOG.error("Could not get resource file {}", resourceFileName);
-            throw new RuntimeException(e);
+            return Optional.of(tempResourceFile);
+        } catch (Exception e) {
+            LOG.error("Could not get resource file {}", resourceFileName, e);
+            return Optional.empty();
+        } finally {
```

Comment:
You could alternatively do this without the finally, i.e., just add the `tempResoureFile.deleteOnExit()` after line 596/593.